CODEOWNERS

```diff
@@ -6,3 +6,6 @@
 # Part of the frontend that mobile developper should review
 src/external_app/ @bgoncal @TimoPtr
 test/external_app/ @bgoncal @TimoPtr
+
+# Part of the frontend that mobile developper and designer should review
+src/resources/theme/color/ @bgoncal @marcinbauer85 @TimoPtr @wendevlin
```

Comment:
This should work on team basis, not on a per person / account basis.

../Frenck

Reply:
I'll update it once we have the teams.

Reply:
For this I think it's fair that mobile team review the change not only being notified since we are discussing about how to sync with the mobile application. So it fit with the purpose of codeowner. As discuss on slack for @marcinbauer85 it's another topic.

The Android and iOS team contains ppl that probably are not interested into this topic. I was hoping to use the OHF teams  if we have them at some point (something like OHF mobile or Android/iOS it's up to you). 

Also nothing is set in stone, so it can be adjust but we needed something to keep us in sync especially in the early stage of the implementation of the design system. 

I don't trust manual process but that only be me...

Reply:
I think we should come to a decission and then implement it. I'll revert the changes.

Would it be possible to integrated it to our home assistant bot?